hack/.golint_failures

```diff
@@ -472,6 +472,7 @@ staging/src/k8s.io/apiextensions-apiserver/pkg/cmd/server
 staging/src/k8s.io/apiextensions-apiserver/pkg/controller/finalizer
 staging/src/k8s.io/apiextensions-apiserver/pkg/controller/status
 staging/src/k8s.io/apiextensions-apiserver/pkg/features
+staging/src/k8s.io/apiextensions-apiserver/pkg/openapi
```

Comment:
why not fix it right away?

Reply:
golint was complaining about how we name the conversion functions with underscore: `Convert_v1beta1_JSONSchemaProps_To_spec_Schema` https://github.com/kubernetes/kubernetes/pull/67205/commits/6dca23fc44e4f9d9fb3bcb0227126db6f46e27eb#diff-19681f6fc979e9acc3ee091c79e3a180R30